contracts/protocol/facets/ExchangeHandlerFacet.sol

```diff
@@ -808,17 +782,7 @@ contract ExchangeHandlerFacet is IBosonExchangeHandler, BuyerBase, DisputeBase {
                 }
             }
 
-            if (transferFailed) {
-                // Raise a dispute if caller is a contract
-                if (sender.isContract()) {
-                    raiseDisputeInternal(_exchange, _voucher, seller.id);
-                } else {
-                    // Revoke voucher if caller is an EOA
-                    revokeVoucherInternal(_exchange);
-                    // N.B.: If voucher was revoked because transfer twin failed, then voucher was already burned
-                    shouldBurnVoucher = false;
-                }
-            }
+            if (transferFailed) raiseDisputeInternal(_exchange, _voucher, seller.id);
```

Comment:
can you put this call to raiseDisputeInternal on line 770 and remove transferFailed variable pls

Reply:
Good point, nice one.

Changed.